pkg/apiserver/registries/registries.go

```diff
@@ -68,6 +69,25 @@ func RegistryImageBlob(request *restful.Request, response *restful.Response) {
 		return
 	}
 
+	// default use ssl
```

Comment:
Is there a protocol check process in the creation？
Validation during creation easily show error messages

Reply:
do you mean validation during creating secret? but here also will use SSL by default if not have protocol.

Reply:
Yes.
Because the validation of the secret creating process is now based on the docker daemon.
If the user configures the insecure registry in daemon, it may cause addresses without a protocol to be converted to no-ssl requests during creating validation.

Reply:
maybe we could add some descriptions to prompt users to use SSL by default.
@minghaoye

Reply:
@soulseen Not all users will see the prompt, and it may cause user confusion. This issue should be addressed at the API level.